coordinator/store/stdstore/stdstore.go

```diff
@@ -122,6 +126,19 @@ func (s *StdStore) SetEncryptionKey(encryptionKey []byte) error {
 	return s.sealer.SetEncryptionKey(encryptionKey)
 }
 
+func (s *StdStore) beginTransaction() (*StdTransaction, error) {
```

Comment:
Moving that into a private function for style reasons?

Reply:
Moved to private so we can work on `*StdTransaction` directly instead of the `store.Transaction` interface